src/test/java/ibm/jceplus/junit/openjceplusfips/TestDSASignature.java

```diff
@@ -1,45 +1,23 @@
 /*
- * Copyright IBM Corp. 2023
+ * Copyright IBM Corp. 2023, 2024
  *
  * Licensed under the Apache License 2.0 (the "License").  You may not use
  * this file except in compliance with the License.  You can obtain a copy
  * in the file LICENSE in the source distribution.
  */
 
 package ibm.jceplus.junit.openjceplusfips;
+import ibm.jceplus.junit.base.BaseTestDSASignature;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
 
-import junit.framework.Test;
-import junit.framework.TestSuite;
+@TestInstance(Lifecycle.PER_CLASS)
+public class TestDSASignature extends BaseTestDSASignature {
 
-public class TestDSASignature extends ibm.jceplus.junit.base.BaseTestDSASignature {
-
-    //--------------------------------------------------------------------------
-    //
-    //
-    static {
+    @BeforeAll
+    public void beforeAll() {
         Utils.loadProviderTestSuite();
+        setProviderName(Utils.TEST_SUITE_PROVIDER_NAME);
     }
-
-    //--------------------------------------------------------------------------
-    //
-    //
-    public TestDSASignature() {
-        super(Utils.TEST_SUITE_PROVIDER_NAME);
-    }
-
-    //--------------------------------------------------------------------------
-    //
-    //
-    public static void main(String[] args) throws Exception {
-        junit.textui.TestRunner.run(suite());
-    }
-
-    //--------------------------------------------------------------------------
-    //
-    //
-    public static Test suite() {
-        TestSuite suite = new TestSuite(TestDSASignature.class);
-        return suite;
-    }
-}
-
+}
```

Comment:
Add new line.

Reply:
New line added in commit bd27b562b46b98c872982261f4a7cb8bb2145955

Additionally a checkstyle rule was added to enforce this in the future.